kahuna/public/js/edits/service.js

```diff
@@ -149,6 +149,17 @@ service.factory('editsService',
         return existingRequestPool.promise;
     }
 
+    function perform(resource, action, originalImage) {
+        const newRequest = resource.perform(action).
+              then(action => resource.get().then(edit => getSynced(originalImage, newImage => matches(edit, newImage))));
```

Comment:
This:
```JSON
{
    "data": {
        "credit": "The Guardian",
        "byline": "James Gorrie",
        "copyright": "Copyright",
        "keywords": []
    }
}
```

We could add the URI to the metadata endpoint as that is what it's modified, does that feel correct or like we're I'm retrofitting a reason to a solution?